components/site/patterns/DeleteConfirm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useUID } from '@twilio-paste/core/uid-library';
+import { Button } from '@twilio-paste/core/button';
+import { Modal, ModalBody, ModalFooter, ModalFooterActions, ModalHeader, ModalHeading } from '@twilio-paste/core/modal';
+import PropTypes from 'prop-types';
+
+export const DeleteConfirm = ({ service, isOpen, setIsOpen, deleteService }) => {
+  const modalHeadingID = useUID();
+  const handleClose = () => setIsOpen(false);
+  const handleConfirm = (serviceObj) => {
+    deleteService(serviceObj);
+    setIsOpen(false);
+  };
+
+  return (
+    <div>
+      <Modal ariaLabelledby={modalHeadingID} isOpen={isOpen} onDismiss={handleClose} size="default">
+        <ModalHeader>
+          <ModalHeading as="h3" id={modalHeadingID}>
+            Delete service
+          </ModalHeading>
+        </ModalHeader>
+        <ModalBody>You&apos;re about to delete {service.friendlyName}. Would you like to proceed?</ModalBody>
```

Comment:
Just curious - is there a reason you're using a character code here?

Reply:
Great question! ^ There are really good reasons for doing this. 

Who knows why? 🍿

Reply:
Has to do with accessibility so screen readers know exactly what the character is?

Reply:
Not quite. There are a few reasons you probably want to use character encoding:

- Some characters are reserved in code to mean specific things. Think ", &, <, > as examples. These are safer to encode as `&quot;`, `&amp;`, `&lt;` and `&gt;` because you are explicitly telling the browser, this is not the start of an HTML tag, it's not the start or end of a string... etc.
- Sometimes its just difficult to type, for example `&copy;` &copy; `&trade;` &trade;
- Other times it's for typography reasons. Where apostrophes `'` vs `&rsquo;`, quotes `"` vs `&ldquo;`, and dashes `-` vs `&mdash;` are more stylistic
  - it's != it&rsquo;s
  - "This is a quote" != &ldquo;This is a quote&rdquo;
- Finally for internationalization, some languages don't share the same character sets so you might what to be explicit rather than rely on unicode

This is certainly not the most perfect answer.